neo/Helper.cs

```diff
@@ -320,5 +320,34 @@ public static IConfigurationRoot LoadConfig(string config)
                 .AddJsonFile(configFile, true)
                 .Build();
         }
+
+        public static T StrictEnum<T>(int value) where T : Enum
+        {
+            T state = (T)Enum.ToObject(typeof(T), value);
+            if (!EnumValueCache<T>.definedValues.Contains(state))
+                throw new FormatException();
+            return state;
+        }
+
+        public static T StrictEnum<T>(byte value) where T : Enum
+        {
+            T state = (T)Enum.ToObject(typeof(T), value);
+            if (!EnumValueCache<T>.definedValues.Contains(state))
+                throw new FormatException();
+            return state;
+        }
+
+        public static T StrictEnumParse<T>(string value) where T : Enum
+        {
+            T state = (T)Enum.Parse(typeof(T), value);
```

Comment:
We should define if we allow or not `ignoreCases` 
My vote: yes

Reply:
I removed `StrictEnumParse` completely.  It was replacing `Enum.Parse`, but that already fails if the entry doesn't exist in the Enum.